src/mobu/services/github_ci/ci_manager.py

```diff
@@ -0,0 +1,336 @@
+"""Manager for running non-continuous business."""
```

Comment:
Could you unpack "non-continuous" a bit more here?

Reply:
Oof, yep, that terminology is left over from when I was implementing this and *I* didn't quite know what it meant either